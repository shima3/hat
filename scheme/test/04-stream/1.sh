#!/bin/bash
$* -e main examples/stream.sch
