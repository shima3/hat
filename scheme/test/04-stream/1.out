e=0
e=1
e=1
e=2
e=3
e=5
